pkgs/development/tools/ocaml/ocamlformat/default.nix

```diff
@@ -25,11 +56,49 @@ with ocamlPackages; buildDunePackage rec {
     fix
     menhir
   ];
+in
+
+rec {
```

Comment:
Can you link your skript here so we can decide in future, which versions of ocamlformat to keept?